### PR TITLE
fix run_remote with strong quoting

### PR DESCRIPTION
**Pull request by lightmark**
add \' pair to avoid bash expanding before run remotely

### CODE REVIEW

File: tools/regression_test.sh [hunk 1]
```diff
@@ -127,7 +127,7 @@ function main {
       tmp=$DB_PATH
       DB_PATH=$ORIGIN_PATH
       test_remote "test -d $DB_PATH"
-      if [[ $? -ne 0 ]] || [[ $(run_remote 'echo $(( $(date +"%s") - $(stat -c "%Y" '"$DB_PATH"') ))') -gt "604800" ]]; then
+      if [[ $? -ne 0 ]] || [[ $(run_remote \''echo $(( $(date +"%s") - $(stat -c "%Y" '"$DB_PATH"') ))'\') -gt "604800" ]]; then
```
Comment (ajkr) [inline tools/regression_test.sh]: i think you want this in test_remote. Otherwise it'll work only conditionally depending on $REMOTE_USER_AT_HOST.

Comment (sagar0) [inline tools/regression_test.sh]: IMO, I don't think you should be using single quotes here. Better to embed the whole thing in double quotes: 
http://unix.stackexchange.com/questions/118433/quoting-within-command-substitution-in-bash .

Comment (lightmark) [inline tools/regression_test.sh]: weak quoting will auto expand variable when I don't want it.

Comment (lightmark) [inline tools/regression_test.sh]: yes

**Pull request closed without merging**